src/api/app/views/webui/attribute/_attributes.html.haml

```diff
@@ -14,7 +14,10 @@
           %td{ class: "#{attribute.namespace}-#{attribute.name}" }
             = attribute.fullname
           %td{ class: "#{attribute.namespace}-#{attribute.name}-values" }
-            = attribute.values.map(&:value).join(', ')
+            %ul
+            - attribute.values.each do |v|
```

Comment:
what about calling this value instead of v in aim of readability? :pray:

Reply:
I can read vs, where is your problem with reading vs?

Reply:
I can also read vs... but it doesn't tell me what it is... while `value` is completely clear...

Reply:
And what would value.value mean to you? Can we agree to disagree?

Reply:
`value.value`? :confused: